### PR TITLE
adding more documentation

### PR DESCRIPTION
**Pull request by lewisc**
trying to give regression test readme a little more info.

### CODE REVIEW

File: regression-tests/README.md [hunk 1]
```diff
@@ -14,5 +15,12 @@ There are two environment variables that need to be set; `SPARKTK_HOME` and
 `PYTHONPATH`. I would recommend setting them in your shell rc file (.bashrc for
```
Comment (blbarker) [inline regression-tests/README.md]: Remove first person singular

File: regression-tests/README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 1. First build the source code, this requires maven to be installed. Run
    `mvn install` at the top level of this repo.
 2. Run `./run_tests.sh` in this folder
+NOTE: THIS WILL DELETE ALL EXISTING DATASETS AND READD THE CONTENTS OF DATASETS
```
Comment (blbarker) [inline regression-tests/README.md]: typo,  and I'm not sure what the last phrase means

File: regression-tests/README.md [hunk 3]
```diff
@@ -14,5 +15,12 @@ There are two environment variables that need to be set; `SPARKTK_HOME` and
...
-`SPARKTK_HOME` needs to be set to `$PWD/../core/target` from this directory
-`PYTHONPATH` needs to be set to `$PWD/sparktkregtests` from this directory
+`SPARKTK_HOME` needs to be set to `<PATH TO SPARK-TK>/sparktk-core/target`
+
+`PYTHONPATH` needs to be set with both pyspark and the spark-tk regression suite libraries
+to `<PATH TO SPARK-TK>/regression-tests:/opt/cloudera/parcels/CDH/lib/spark/python/:PYTHONPATH`
+
+In addition you need to make sure your datasets are up to date, to do this you
+run the `install_datasets.sh` file out of the automation folder.
+NOTE: THIS WILL DELETE ALL EXISTING DATASETS AND READD THE CONTENTS OF DATASETS
```
Comment (blbarker) [inline regression-tests/README.md]: copy paste typo

File: regression-tests/README.md (hunk 2 above)
Comment (lewisc) [inline regression-tests/README.md]: Re-Add, as in add again.

**Pull request merged**